autogpt_platform/frontend/src/app/(platform)/signup/useSignupPage.ts

```diff
@@ -15,6 +15,7 @@ export function useSignupPage() {
   const router = useRouter();
   const [isLoading, setIsLoading] = useState(false);
   const [isGoogleLoading, setIsGoogleLoading] = useState(false);
+  const isProdEnv = process.env.NODE_ENV === "production";
```

Comment:
We have helpers for this for local dev and prod as an enum because identifying them isn’t always trivial

Reply:
I have refactored to use `getBehaveAs()` 👍🏽